README.md

```diff
@@ -11,16 +11,11 @@ A unified publishing application for content on GOV.UK
 
 ## Technical documentation
 
-This is a Ruby on Rails application. You can use the GOV.UK Docker environment to run the application and its tests with all the necessary dependencies; **see the [README](https://github.com/alphagov/govuk-docker#usage) for more details**.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-### Dependencies
+**Use GOV.UK Docker to run any commands that follow.**
 
-- [postgresql][] - provides a backing database
```

Comment:
I agree that this section is often incomplete or out of date, but I do find it useful to know what database an application is using, as it affects the app console syntax I'm able to use (Postgres vs Mongo). I'll sometimes check the repo to figure out which one it's using before I query (though more often than not, I'll just use trial and error in my queries).

It's a shame that the `gds govuk connect app-console` command doesn't expose what the DB is, which would mitigate this concern.

Reply:
Yes, it was tempting to leave this section in. But it's really a symptom of a bigger problem: if there's a need to know about these dependencies, then the repo should have actual documentation about their use. In the case you describe, it sounds like there's a need for "common DB queries" documentation. Simply calling out "it's a mongo thing" only provides a relatively small piece of information on the journey to getting the required data.

I don't think it's going to cause a major issue: someone can easily check `config/` to see what DB is in use. By removing it, we make room for having useful, actionable documentation about it.

Reply:
I think the right place to make a change would be in [govuk-connect](https://github.com/alphagov/govuk-connect), making it state which DB is in use when you open a console. So happy to approve this section being removed 👍